src/hmpps/components/legal-information/sub-navigation/template.njk

```diff
@@ -0,0 +1,83 @@
+{% from "moj/components/sub-navigation/macro.njk" import mojSubNavigation %}
+
+{% set environment = environment if environment else 'dev' %}
+
+{% set environmentHrefs = {
+    local: {
+        overview: 'http://localhost:3000/prisoner/' + prisonNumber + '/overview',
+        courtCases: 'http://localhost:3000/person/' + prisonNumber,
+        adjustments: 'http://localhost:3000/' + prisonNumber,
+        releaseDates: 'http://localhost:3000/?prisonId=' + prisonNumber,
+        documents: 'http://localhost:3000/prisoner/' + prisonNumber + '/documents'
+    },
+    dev: {
+        overview: 'https://custody-manager-dev.hmpps.service.justice.gov.uk/prisoner/' + prisonNumber + '/overview',
+        courtCases: 'https://remand-and-sentencing-dev.hmpps.service.justice.gov.uk/person/' + prisonNumber,
+        adjustments: 'https://adjust-release-dates-dev.hmpps.service.justice.gov.uk/' + prisonNumber,
+        releaseDates: 'https://calculate-release-dates-dev.hmpps.service.justice.gov.uk/?prisonId=' + prisonNumber,
+        documents: 'https://custody-manager-dev.hmpps.service.justice.gov.uk/prisoner/' + prisonNumber + '/documents'
+    },
+    pre: {
+        overview: 'https://custody-manager-preprod.hmpps.service.justice.gov.uk/prisoner/' + prisonNumber + '/overview',
+        courtCases: 'https://remand-and-sentencing-preprod.hmpps.service.justice.gov.uk/person/' + prisonNumber,
+        adjustments: 'https://adjust-release-dates-preprod.hmpps.service.justice.gov.uk/' + prisonNumber,
+        releaseDates: 'https://calculate-release-dates-preprod.hmpps.service.justice.gov.uk/?prisonId=' + prisonNumber,
+        documents: 'https://custody-manager-preprod.hmpps.service.justice.gov.uk/prisoner/' + prisonNumber + '/documents'
+    },
+    prod: {
+        overview: 'https://custody-manager.hmpps.service.justice.gov.uk/prisoner/' + prisonNumber + '/overview',
+        courtCases: 'https://remand-and-sentencing.hmpps.service.justice.gov.uk/person/' + prisonNumber,
+        adjustments: 'https://adjust-release-dates.hmpps.service.justice.gov.uk/' + prisonNumber,
+        releaseDates: 'https://calculate-release-dates.hmpps.service.justice.gov.uk/?prisonId=' + prisonNumber,
+        documents: 'https://custody-manager.hmpps.service.justice.gov.uk/prisoner/' + prisonNumber + '/documents'
+    }
+} %}
```

Comment:
Would it be possible to make these configurable externally? Or at least provided as a value to the macro?

Reply:
Else if it does move there is a dependency on updating and up versioning this great library...
There are swings and roundabout to both though as this can be the source of truth

Reply:
These are the defaults if you don't pass in a URL. If you pass in a navigation structure as follows:

```
{% set navigation = {
    activeSubNav: 'overview',
    overview: {
        href: '#overview'
    },
    courtCases: {
        href: '#court-cases'
    },
    adjustments: {
        href: '#adjustments'
    },
    releaseDates: {
        href: '#release-dates'
    },
    documents: {
        href: '#documents'
    }
} %}
```

It will use the supplied hrefs instead of the environment defaults listed above

Reply:
Ah yes I see that now, apologies, i'll look harder next time.
Good stuff